### PR TITLE
Eigenmiao.Rickrack version 2.5.24.0

### PR DESCRIPTION
**Pull request by eigenmiao**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/68944)

### CODE REVIEW

File: manifests/e/Eigenmiao/Rickrack/2.5.24.0/Eigenmiao.Rickrack.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Eigenmiao.Rickrack
+PackageVersion: 2.5.24.0
```
Comment (ItzLevvie) [inline manifests/e/Eigenmiao/Rickrack/2.5.24.0/Eigenmiao.Rickrack.installer.yaml:5]:
```suggestion
PackageVersion: 2.5.24
```
This must match Control Panel.

File: manifests/e/Eigenmiao/Rickrack/2.5.24.0/Eigenmiao.Rickrack.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Eigenmiao.Rickrack
+PackageVersion: 2.5.24.0
```
Comment (ItzLevvie) [inline manifests/e/Eigenmiao/Rickrack/2.5.24.0/Eigenmiao.Rickrack.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.5.24
```
This must match Control Panel.

File: manifests/e/Eigenmiao/Rickrack/2.5.24.0/Eigenmiao.Rickrack.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: Eigenmiao.Rickrack
+PackageVersion: 2.5.24.0
```
Comment (ItzLevvie) [inline manifests/e/Eigenmiao/Rickrack/2.5.24.0/Eigenmiao.Rickrack.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 2.5.24
```
This must match Control Panel.

File: manifests/e/Eigenmiao/Rickrack/2.5.24.0/Eigenmiao.Rickrack.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Eigenmiao.Rickrack
+PackageVersion: 2.5.24.0
```
Comment (ItzLevvie) [inline manifests/e/Eigenmiao/Rickrack/2.5.24.0/Eigenmiao.Rickrack.yaml:5]:
```suggestion
PackageVersion: 2.5.24
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to update the version folder from `2.5.24.0` to `2.5.24`.

For example: `manifests/e/Eigenmiao/Rickrack/2.5.24.0` to `manifests/e/Eigenmiao/Rickrack/2.5.24`

**Review by ItzLevvie — APPROVED** [review]

**Comment by eigenmiao** [comment]
> You'll also need to update the version folder from `2.5.24.0` to `2.5.24`.
> 
> For example: `manifests/e/Eigenmiao/Rickrack/2.5.24.0` to `manifests/e/Eigenmiao/Rickrack/2.5.24`

Ok, thanks, I will revise it and re-submit it later.

**Pull request closed without merging**